s3_website.yml

```diff
@@ -31,6 +31,7 @@ redirects:
   briefcase-guide/index.html: briefcase-intro
   briefcase-vs-aggregate/index.html: briefcase-intro
   briefcase-and-aggregate/index.html: briefcase-intro
+  central-datasets/index.html: central-entities
```

Comment:
@matthew-white Does this redirect address your concern?

Reply:
Ah I didn't notice that, excellent. ✨